streams/do.m3u

```diff
@@ -45,6 +51,16 @@ https://ss3.domint.net:3150/c25_str/canal25/playlist.m3u8
 https://cdn5.streamgato.us:3371/live/canal56live.m3u8
 #EXTINF:-1 tvg-id="CanalCatorce.do",Canal Catorce (720p)
 https://tv.livestreaminggroup.info:3052/live/catorcetvlive.m3u8
+#EXTINF:-1 tvg-id="CanaldelSol.do",Canal del Sol 6 (720p)
+https://5790d294af2dc.streamlock.net/canal6/canal6/playlist.m3u8
+#EXTINF:-1 tvg-id="CanaldelSol.do",Canal del Sol 65 (720p)
```

Comment:
Potential channel conflict with Canal del Sol 65, especially if EPG is available for both of them.